Makefile

```diff
@@ -129,15 +131,23 @@ statusgo-library: ##@cross-compile Build status-go as static library for current
 
 docker-image: ##@docker Build docker image (use DOCKER_IMAGE_NAME to set the image name)
 	@echo "Building docker image..."
-	docker build --file _assets/build/Dockerfile --build-arg "build_tags=$(BUILD_TAGS)" --build-arg "build_flags=$(BUILD_FLAGS)" . -t $(DOCKER_IMAGE_NAME):latest
+	docker build --file _assets/build/Dockerfile . \
+		--build-arg "build_tags=$(BUILD_TAGS)" \
+		--build-arg "build_flags=$(BUILD_FLAGS)" \
+		-t $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_CUSTOM_TAG) \
+		-t $(DOCKER_IMAGE_NAME):latest
 
 bootnode-image:
 	@echo "Building docker image for bootnode..."
-	docker build --file _assets/build/Dockerfile-bootnode . -t $(BOOTNODE_IMAGE_NAME):latest
-
-docker-image-tag: ##@docker Tag DOCKER_IMAGE_NAME:latest with a tag following pattern $GIT_SHA[:8]-$BUILD_TAGS
-	@echo "Tagging docker image..."
-	docker tag $(DOCKER_IMAGE_NAME):latest $(DOCKER_IMAGE_NAME):$(shell BUILD_TAGS="$(BUILD_TAGS)" ./_assets/ci/get-docker-image-tag.sh)
+	docker build --file _assets/build/Dockerfile-bootnode . \
+		-t $(BOOTNODE_IMAGE_NAME):$(DOCKER_IMAGE_CUSTOM_TAG) \
+		-t $(BOOTNODE_IMAGE_NAME):latest
+
+push-docker-images: docker-image bootnode-image
```

Comment:
This method may bring a lot of confusion if run by someone who forgets to update the branch or runs it from a different branch than `develop`. I would recommend pushing images with `$(DOCKER_IMAGE_CUSTOM_TAG)` tags and have a dedicated command to push `latest` tag which verifies it's not `develop` and it's up-to-date.

Reply:
That's fair, will change it.

Reply:
Although, what do you mean when you say "verifies it's not `develop`"? You mean `latest` tag should never be updated form the `develop` branch?

Reply:
Oh sorry. I meant "verifies it IS `develop`". So `latest` tag should actually be only created from the main branch which in our case is `develop`.

Reply:
Added a check.